src/lib/packager/product_patterns.rb

```diff
@@ -0,0 +1,141 @@
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# ------------------------------------------------------------------------------
+#
+
+require "yast"
+
+module Packager
```

Comment:
Is that always needed? E.g. bootloader uses [Bootloader](https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/device_map_dialog.rb#L10), nfs-client [Fstab::TSort](https://github.com/yast/yast-nfs-client/blob/master/src/lib/fstab/tsort.rb#L4), IIRC the convention is to use the package name.

@jreidinger ?